pkg/capabilities/consensus/ocr3/capability.go

```diff
@@ -92,22 +102,14 @@ func (o *capability) HealthReport() map[string]error {
 }
 
 type workflowConfig struct {
-	AggregationMethod string      `mapstructure:"aggregation_method"`
-	AggregationConfig *values.Map `mapstructure:"aggregation_config"`
-	Encoder           string      `mapstructure:"encoder"`
-	EncoderConfig     *values.Map `mapstructure:"encoder_config"`
-}
-
-func newWorkflowConfig() *workflowConfig {
-	return &workflowConfig{
-		EncoderConfig:     values.EmptyMap(),
-		AggregationConfig: values.EmptyMap(),
```

Comment:
@HenryNguyen5 Did anything break when you removed this? IIRC I added this to ensure the map was properly initialized (with its internal map[string]any)

Reply:
Surprisingly not? Unless somehow it'd not caught by the test suite we currently have. I made sure all tests were passing as I was refactoring